numpyro/distributions/continuous.py

```diff
@@ -67,6 +66,117 @@
 )
 
 
+class AsymmetricLaplace(Distribution):
+    arg_constraints = {
+        "loc": constraints.real,
+        "scale": constraints.positive,
+        "asymmetry": constraints.positive,
+    }
+    reparametrized_params = ["loc", "scale"]
```

Comment:
Looking at the sampler, I think all params are reparameterized (with base dists are exponentials).